tests/test_volume.py

```diff
@@ -257,16 +256,25 @@ def testCnSymmetricVolume(self):
                 ref = vol[0, selection]
                 rot = rot_vol[0, selection]
 
-                # Assert that rotated volume is within .5% of original volume.
-                self.assertTrue(np.amax(abs(rot - ref) / ref) < 0.005)
+                # Assert that voxel-wise the rotated volume has a maximum error of <2%
+                # and a mean error of <0.1%.
+                self.assertTrue(np.amax(abs(rot - ref) / ref) < 0.02)
+                self.assertTrue(np.mean(abs(rot - ref) / ref) < 0.001)
+
+    def testAsymmetricVolume(self):
```

Comment:
Why are these test not in `test_synthetic_volume`?

Reply:
moved

Reply:
Just want to mention that this was moved and renamed to `testCompactSupport` in `test_synthetic_volume.py`. We are now testing the compact support on all volume but Legacy.

Reply:
Awesome. Starting to make use of those classes :)